### PR TITLE
tidy/features: fix checking of lang features

### PR DESCRIPTION
**Pull request by tamird**
r? @brson 

See the commit messages - the actual "fixes" here are strawmen; I'm happy to adjust them if you have suggestions.

### CODE REVIEW

File: src/libstd/macros.rs [hunk 1]
```diff
@@ -286,7 +286,6 @@ pub mod builtin {
     /// // fn concat_idents!(new, fun, name) { } // not usable in this way!
     /// # }
     /// ```
-    #[unstable(feature = "concat_idents", issue = "29599")]
```
Comment (tamird) [inline src/libstd/macros.rs]: Done.

**Comment by tamird** [comment]
@brson ping?

**Comment by brson** [comment]
@tamird Hi! Sorry for not seeing this earlier. This looks great. Do you mind adding an assert that some lang features are found, so that this doesn't regress again in the future?

**Comment by tamird** [comment]
Yep! Done.

**Pull request merged**